modules/fx/arrow-fx/src/test/kotlin/arrow/fx/predef.kt

```diff
@@ -13,9 +15,11 @@ import java.util.concurrent.ThreadFactory
 import java.util.concurrent.ThreadPoolExecutor
 import java.util.concurrent.TimeUnit
 
-fun <A> EQ(EQA: Eq<A> = Eq.any(), timeout: Duration = 60.seconds): Eq<Kind<ForIO, A>> = Eq { a, b ->
-  arrow.core.Option.eq(arrow.core.Either.eq(Eq.any(), EQA)).run {
-    a.fix().attempt().unsafeRunTimed(timeout).eqv(b.fix().attempt().unsafeRunTimed(timeout))
+fun <A> EQ(EQA: Eq<A> = Eq.any(), timeout: Duration = 5.seconds): Eq<Kind<ForIO, A>> = Eq { a, b ->
+  arrow.core.Either.eq(Eq.any(), EQA).run {
+    IO.applicative().mapN(a.attempt(), b.attempt()) { (a, b) ->
+      a.eqv(b)
+    }.waitFor(timeout).unsafeRunSync()
```

Comment:
This `unsafeRunSync` is reentrant, right?

Reply:
'pparently not hahaha Why 60s timeout, and not 5, or 10? When I was testing locally these tests took much longer than everything else on fx.